restarter/hot-restarter.py

```diff
@@ -92,8 +143,8 @@ def sigchld_handler(signum, frame):
       kill_all_and_exit = True
 
   if kill_all_and_exit:
-    print "Due to abnormal exit, force killing all child processes and exiting"
-    force_kill_all_children()
+    print "Due to abnormal exit, terminating all child processes and exiting"
+    term_all_children()
```

Comment:
Not sure, @danielhochman for comment.

Reply:
@mpuncel I'm a little late here but I would probably switch this back to `term_all_children()` since I do think in the abnormal case we should kill as fast as possible and allow the supervisor to restart. Do you mind doing a follow up?

Reply:
no problem! https://github.com/envoyproxy/envoy/pull/2640/files